assets/section-main-product.css

```
@@ -1203,3 +1203,56 @@ a.product__text {
   margin-left: 1.2rem;
   flex-shrink: 0;
 }
+
+/* Icon with text */
+.icon-large .icon {
+  width: 3rem;
+  height: 3rem;
+  margin-bottom: 1rem;
+}
+
+.image-large {
+  height: 3rem;
+  margin-bottom: 1rem;
+}
+
+.icon-large {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.icon-small .icon {
+  min-width: 2rem;
+  min-height: 2rem;
+  margin-right: 1rem;
+}
+
+.icon-small {
+  display: flex;
+  align-items: center;
+  margin: 0 0 1rem 0;
+}
+
+.image-small {
+  width: 2rem;
+  margin-right: 1rem;
+}
+
+.icon-container-vertical {
+  padding-top: 1.5rem;
+}
+
+.icon-container-horizontal--space-between {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-top: 2rem;
+}
+
+.icon-container-horizontal--space-evenly {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  padding-top: 2rem;
+}
```

I think you could simplify the styles/classes here a bit by using a`.icon-container-horizontal` class with all the shared styles and a default `justify-content` value. Then you can use the `--space-evenly` modifier class to override only what's needed. This should allow you to remove an `else` case from the liquid where you conditionally render `evenly` or `between`

```suggestion
.icon-container-horizontal {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  padding-top: 2rem;
}

.icon-container-horizontal--space-evenly {
  justify-content: space-evenly;
}
```